tests/0113-cooperative_rebalance.cpp

```diff
@@ -30,83 +30,306 @@
 #include <map>
 #include <cstring>
 #include <cstdlib>
+#include <assert.h>
 #include "testcpp.h"
 
 /**
- * The beginnings of an integration test for cooperative rebalancing.
- * MH: what i'm currently using to debug with.
+ * MH: what i'm currently using to debug with. Not finished.
```

Comment:
not reviewing this file

Reply:
good, otherwise the number of comments would have at least doubled.